Changes.md

```diff
@@ -13,6 +13,7 @@ Improvements
 Fixes
 -----
 
+- GafferTractor: evaluate 'tags' and 'services' using correct context
```

Comment:
Hey Alex, sorry to nit-pick but we keep `Changes.md` in the [`<insert proper grammatical name>`] past tense, and as a sentence in terms of case (check the entry below this one, notably the space either side of the colon, capital for `E` and the full stop at the end).  The general rule is "describe what you did" rather than "what do we do now".

Reply:
Also, imagine the comment is for the end user, so it might be worth expanding on what is different about the context now.

Reply:
Update the commit to: GafferTractor: Fixed evaluation of 'tag' and 'service' plugs on Task nodes. Previously, these plugs were evaluated in the default context, which prevented one from using custom context variables (e.g. from Wedge node) to compute tags or service keys dynamically.